### PR TITLE
Fix: uiSchema structs optimization

### PR DESCRIPTION
**Pull request by wangbow**
### Description of your changes

Fixes #337 uiSchema structs optimization

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### Special notes for your reviewer

image
image
image

### CODE REVIEW

File: src/extends/InnerGroup/index.less [hunk 1]
```diff
@@ -4,7 +4,7 @@
   .ui-schema-container {
     display: flex;
     .next-select {
-      width: 100%;
+      width: 178px !important;
```
Comment (barnettZQG) [inline src/extends/InnerGroup/index.less:7]: 为啥是178？

File: src/extends/Structs/index.tsx [hunk 2]
```diff
@@ -54,31 +58,58 @@ class StructItem extends React.Component<StructItemProps> {
...
-          <Icon
-            type="ashbin"
-            onClick={() => {
-              if (this.props.delete) {
-                this.props.delete(this.props.id);
-              }
-            }}
-          />
-        </div>
+        <If condition={param && param.length > 3}>
```
Comment (barnettZQG) [inline src/extends/Structs/index.tsx:61]: 应该根据 uiSchemas 的长度。

Comment (wangbow) [reply src/extends/Structs/index.tsx]: done

File: src/extends/InnerGroup/index.less (hunk 1 above)
Comment (wangbow) [reply src/extends/InnerGroup/index.less]: done
image
image

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**